tests/functional/z_cluster/cluster_expansion/test_resize_osd.py

```diff
@@ -296,3 +299,20 @@ def test_resize_osd_for_large_diff(self, size_to_increase):
         self.prepare_data_before_resize_osd()
         resize_osd(self.new_storage_size)
         self.verification_steps_post_resize_osd()
+
+    @tier1
+    @tier4
+    @black_squad
+    @pytest.mark.last
+    @polarion_id("OCS-5800")
+    def test_ui_storage_size_post_resize_osd(self, setup_ui_session):
+        """
+        Test the new total storage size is reflected in the UI post resize osd
+
+        """
+        if config.RUN["resize_osd_count"] < 1:
+            pytest.skip(
+                "No resize osd has been performed in the current test run. "
+                "The test should run only post resize osd"
+            )
+        check_storage_size_is_reflected_in_ui()
```

Comment:
IIUC, the scenario seems more like a verification step. Instead of creating a new test, do you think we should include it as part of the verification step after every osd resize functional test?

Reply:
I also thought of including this in the verification steps, but there is a small problem. We want to separate the cli checks from the UI checks. That way, the black squad members can focus on the UI scenario, and the brown squad members can concentrate on the main CLI scenario.

Reply:
agree with Itzhak on behalf of the Black Squad. 
Additionally, the separation will bring more stability and more accurate results delivering only UI or OSD-related issues

Reply:
Ok, I'm fine if this has already been discussed between the squads. I see that we are verifying whether the osd size is correctly reflected after resizing the osd via cli way for each test [here](https://github.com/yitzhak12/ocs-ci/blob/aae62925970a0498e9d942b85de81c8623b20f4a/tests/functional/z_cluster/cluster_expansion/test_resize_osd.py#L168), so, it should be okay.